Fix theme to work with latest GNOME 3.20.2

polter-rnd opened the pull request:
Fixed GTK2 theme using latest available upstream theme and gnome-shell theme to look like default Adwaita.

abihf commented:
Hi @polter-rnd! thanks for your update.

I pick your Gtk2 changes, but I think I can't accept your gnome-shell. Please don't edit .css file directly, because it will be overwritten by scss compiler. Try to edit .scss files then run scss compiler.

polter-rnd commented:
Fixed this. I've changed font to Cantarell and panel background to black because this theme is expected to be identical to Adwaita, but more compact :) Anyway, this changes can be made individually by user.
